.rubocop.yml

```diff
@@ -64,6 +64,10 @@ Style/HashSyntax:
   Exclude:
     - 'docs/lib/govuk_tech_docs/open_api/renderer.rb'
 
+Style/MapIntoArray:
+  Exclude:
+    - 'docs/lib/govuk_tech_docs/open_api/renderer.rb'
```

Comment:
Is it painful to fix this instead of excluding it? I'd rather abide by these cops where possible.

Reply:
🤔 I think this was going to be tricky and there are no tests which made me reluctant to change anything 😬

Reply:
Fair, lets 🦵🏼 the can